torch/csrc/inductor/inductor_ops.cpp

```diff
@@ -22,10 +22,29 @@ Tensor _mm_plus_mm(
   return out;
 }
 
+// Similar to as_strided with the following differences
```

Comment:
Probably worth to also mention these in the `ir.ReinterpretView` class's comment.